### DISCUSSION TITLE
Question: Adding OpenSSL in python3.8 Dockerfile

### DISCUSSION THREAD

**Discussion by akempkens** [tiangolo/uwsgi-nginx-docker#216] [Questions] [answered]
I'm using GRPC for our Microservices and established an openssl based certificate within the communication of the frontend container with nginx+flask towards these backend containers.

The issue is now that 
1) the normal python3.8 Docker does not come with openssl included
2) the alpine version comes with openssl, however the compiler for GRPC (CC and C++) are not included

As this container is the base for the -flask version I ended up with adding openssl into the install-nginx-debian.sh which works. However I wonder if this is the intended way to do it, or if you have a better proposal.

Thx for feedback

**Comment by tiangolo** [comment, marked as answer]
I think you can probably add it in a Docker `RUN apt update && apt install ...` instruction in your `Dockerfile`.
